core/server/models/newsletter.js

```diff
@@ -0,0 +1,14 @@
+const ghostBookshelf = require('./base');
+
+const Newsletter = ghostBookshelf.Model.extend({
+    tableName: 'newsletters'
+});
+
+const Newsletters = ghostBookshelf.Collection.extend({
```

Comment:
I think you can remove this, I'm pretty sure this is a pattern we want to move away from

refs https://ghost.slack.com/archives/C02G9E68C/p1631721852044500

Reply:
awesome, thanks!